packages/cascara/src/modules/ActionButton/ActionButton.js

```diff
@@ -20,6 +22,8 @@ const ActionButton = ({
   ...rest
 }) => {
   const { isEditing, onAction, record } = useContext(ModuleContext);
+
+  // @bje we need to decide if we go for content or label here, both makes no sense
```

Comment:
I agree. I was looking at this and thought it was confusing when looking at the comment on the prop type description.

Initially, I was thinking it would make sense for modules to have almost the same shape and you could just switch between them without having to change the shape at all. That might not make sense anymore... at least not for "Action" type modules.

Go ahead and create a separate task for renaming this. We will want to deprecate "label" with a warning for a couple releases, then make sure everything gets migrated in the codebase before we remove that completely.

Reply:
Tasks:

- [ ] Create a task for deprecating `label` prop in modules, we want to stick to `content`

Reply:
FDS-241 was created to track this request. The changes have been included in this PR.